src/ophyd_async/epics/advimba/_vimba.py

```diff
@@ -39,3 +39,38 @@ def __init__(
             name=name,
             config_sigs=config_sigs,
         )
+
+
+class ContAcqVimbaDetector(adcore.ContAcqAreaDetector[ContAcqVimbaController]):
```

Comment:
Is this something we would ever want to use with a Vimba detector? If not, then we probably shouldn't include this class in case we confuse people...

Reply:
We actually do have cases where this would be desirable - specifically in cases where these are monitoring cameras that the beamline wants to just take a snapshot with of some current setup for their logbook.

Reply:
Hmm, but maybe for that case we should be just controlling the file writer rather than the driver, can we discuss in the collab meeting today?